### PR TITLE
Evacutate OSDs simultaneously when removing OSDs

### PR DESCRIPTION
**Pull request by swiftgist**
Signed-off-by: Eric Jackson <swiftgist@gmail.com>
bnc#1141285

-----------------

**Checklist:**
- [x] Added unittests and or functional tests
- [ ] Adapted documentation
- [x] Referenced issues or internal bugtracker
- [ ] Ran integration tests successfully (trigger with "@susebot run teuthology" in a GitHub comment; see the [wiki](https://github.com/SUSE/DeepSea/wiki/Testing) for more information)

### CODE REVIEW

**Comment by swiftgist** [comment]
@susebot run teuthology

File: srv/modules/runners/rebuild.py [hunk 1]
```diff
@@ -65,7 +63,7 @@ def __init__(self, *args, **kwargs):
         self.qrunner = salt.runner.RunnerClient(__master_opts__)
         self.minions = self._minions(*args)
         self.operation = kwargs.get('operation', 'osd.remove')
```
Comment (jschmid1) [inline srv/modules/runners/rebuild.py]: that needs to be removed aswell.. A user could overwrite this attribute

**Comment by jschmid1** [comment]
@susebot run teuthology

File: srv/modules/runners/rebuild.py [hunk 2]
```diff
@@ -64,9 +66,12 @@ def __init__(self, *args, **kwargs):
         __master_opts__['quiet'] = True
         self.qrunner = salt.runner.RunnerClient(__master_opts__)
         self.minions = self._minions(*args)
-        self.operation = kwargs.get('operation', 'osd.remove')
-        self.aggressive = kwargs.get('aggressive', False)
+        self.kwargs = kwargs
         self.skipped = []
+        if 'preserve_ids' in kwargs and kwargs['preserve_ids']:
```
Comment (jschmid1) [inline srv/modules/runners/rebuild.py]: for future reference:

```suggestion
        if kwargs.get('preserve_ids', False): # or whatever the default should be 
```

**Pull request merged**